code/modules/cargo/markets/market_items/clothing.dm

```diff
@@ -79,3 +79,19 @@
 	price_max = CARGO_CRATE_VALUE * 1.5
 	stock_max = 5
 	availability_prob = 70
+
+/datum/market_item/clothing/floortileset
+	name = "Floor-tile Camouflage Uniform"
+	desc = "Thank you for shopping from Camo-J's, our uniquely designed \
+		floor-tile 'NT SCUM' styled camouflage fatigues is the ultimate \
+		espionage uniform used by the very best. Providing the best \
+		flexibility, with our latest Camo-tech threads. Perfect for \
+		risky espionage hallway operations. Enjoy our product!"
```

Comment:
Compared to the other descriptions for market items, this is a little long, plus it's worded like they've already bought it. This description is only shown on the black market UI before you buy it, so you might want to reword so it's a bit more concise as to what it does and who it's from.